hrm-service/unit-tests/permissions/setupCanForRules.test.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable jest/no-standalone-expect */
 import each from 'jest-each';
 import { setupCanForRules } from '../../src/permissions/setupCanForRules';
-import { actionsMaps, User } from '../../src/permissions';
+import { actionsMaps } from '../../src/permissions';
 import { RulesFile } from '../../src/permissions/rulesMap';
 import { workerSid, accountSid } from '../../service-tests/mocks';
+import { user as newUser } from '@tech-matters/twilio-worker-auth';
```

Comment:
I feel like `createXXX` indicates you are doing something permanent, like creating it in your data store or something, rather than just instantiating a local object. I like the function named after the object convention because it's concise & expressive - it's just a constructor that you don't need the `new` keyword for, though it can lead to name clashes, but those don't concern me greatly because they are the exception & easily addressed. 

I prefer the current naming but can go with `newUser` if you have a strong preference

Reply:
Actually, you know what, I'm just going to rename the type and creator func to `twilioUser`, more descriptive anyway

Reply:
The idea that "create" means something permanent in the backend makes total sense to me. 

I think I may have diluted the point I was trying to make by saying that `newUser()` was a verb form when it isn't really. `newUser` is an adjective noun naming convention so it is still, in my opinion, probably not a great choice for naming a function. Perhaps `makeUser()` or `initUser()` or just about any other `verbNoun()` combo would be a better standard?

And to be clear... I'm not harping on this one instance because I think it is a vitally important instance. This is one of those conversations that is more about setting a standard for ourselves and others to follow in the future rather than the specific instance that we are discussing.

Reply:
The reason I say this is standard setting is because I want to be able to look at code in any of our repos, but especially this one, at some point in the future and know that `makeUser()` or `makeSomeObject()` or `generateUser()` or whatever we choose is going to be one of our TS based Crockford Object generators and it will be doing something similar to `new SomeObject()` underneath without having to inspect the code. Since there is a **very** good chance that at some point in the future we'll be assigning the return of one of those functions to a variable that we would prefer to name `user` or `someObject` or `twilioUser`, it probably isn't a great standard to use noun based names for the functions that generates the objects.

Reply:
Searching the codebase for `make[A-Z]` (and the other forms I've listed here) leads me to believe that it could be a good choice since it is the least used pattern throughout our primary repos.

`construct[A-Z]` returns no results and would be more obvious what is happening underneath to folks who are already familiar with JS classes and other more traditional OO languages but it is a bit long winded.

Reply:
I did understand the point you are make, I just don't agree. Generally I think `verbNoun` is a good function naming convention, but I think object creators are a worthy exception. To me the verb is just naming dead weight and doesn't convey any more information that the noun by itself. 

I think when the function is just the noun it reads better when used inline in expressions and object / array literals, which is probably how they get used more often, and IMHO forcing people to give an individual instance of a type a better, more contextual name than just repeating the type name in lower case is actually a good thing!

Reply:
In the spirit of 'a standard I don't like is better than none' I'm fine to not get my way on this, but perhaps it's better to run it by the team (they will probably agree with you 😅 )